src/modules/dargDrop.js

```diff
@@ -0,0 +1,46 @@
+import { tasks, renderTasks } from './todo.js';
+
+let draggedIndex = null;
+
+export function enableDragAndDrop() {
```

Comment:
You have done a good job so far, but as part of good Javascript practices, it is preferred to use arrow functions.